### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by EvaKaul**
- updated used components to V1.3
- restructured whole how to
- new pictures and screenshots

### CODE REVIEW

File: docs/Usage.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+
+- select the tab "My Plant"
+- go to the newly created dashboard
+- select "Create first widget" to start the wizard
+
+1) choose the Heatmap widget type > Continue
+2) enter a widget name and set the KPI calculation period to 1 Minute > Continue
+3) click "Select parameter" and choose the KPI instance for 'Total energy' > Continue
+4) no need to change anything in the general display options > Continue
+5) no need to change anyting in the Heatmap dispaly options > Finish
```
Comment (mattDoerr) [inline docs/Usage.md]: typo: dispaly

File: docs/Usage.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+- go to the dashboard
+- click the settings button and choose "New widget" to start the wizard
+
+1) choose the Diagram widget type > Continue
+2) enter a widget name and set the KPI calculation period to 15 Minutes > Continue
+3) click "New KPI instance" to create an instance based on the KPI type 'Costs energy', enter a name, select the KPI type and link the parameter 'TotalEnergy' to the already existing instance of 'Total Energy'
+
+![Energy_Manager_Line_KPI_Instance_1](/docs/graphics/Energy_Manager_Line_KPI_Instance_1.png)
+
+3) Again click "New KPI instance" to create an instance based on the KPI type 'Costs water', enter a name, selectthe KPI type and link the parameter 'WaterConsumption' to the signal 'waterConsumptionFillingTank' with aggregation 'Last' > Continue
```
Comment (mattDoerr) [inline docs/Usage.md]: typo: selectthe

File: docs/Usage.md [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+Using the Sankey diagram, energy flows can be visualized. The energy flows are displayed as arrows whose width is proportional to the flow rate.
+
+Here we want to display the water and energy consumption as Sankey:
+
+- go to the dashboard
+- click the settings button and choose "New widget" to start the wizard
+
+1) choose the Sankey widget type > Continue
+2) enter a widget name and set the KPI calculation period to 1 Minute > Continue
+3) click "Select parameter" and choose the energy and water parameter (hint: choose them in the order you would like to see them in the diagram), for all set aggregation to 'Last' > Continue
```
Comment (mattDoerr) [inline docs/Usage.md:172]: very good hint

File: docs/Usage.md [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+Here we want to display the KPI value for 'Total energy' as duration curve:
+
+- go to the dashboard
+- click the settings button and choose "New widget" to start the wizard
+
+1) choose the Duration Curve widget type > Continue
+2) enter a widget name and set the KPI calculation period to 1 Minute > Continue
+3) click "Select parameter" and choose the KPI instance for 'Total energy' > Continue
+4) no need to change anything in the general display options > Continue
+5) no need to change anyting in the Heatmap dispaly options > Finish
```
Comment (mattDoerr) [inline docs/Usage.md]: typo: anyting

File: docs/Usage.md [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+3) click "New KPI instance" to create an instance based on the KPI type 'Costs energy', enter a name, select the KPI type and link the parameter 'TotalEnergy' to the already existing instance of 'Total Energy'
+
+![Energy_Manager_Line_KPI_Instance_1](/docs/graphics/Energy_Manager_Line_KPI_Instance_1.png)
+
+3) Again click "New KPI instance" to create an instance based on the KPI type 'Costs water', enter a name, selectthe KPI type and link the parameter 'WaterConsumption' to the signal 'waterConsumptionFillingTank' with aggregation 'Last' > Continue
+
+![Energy_Manager_Line_KPI_Instance_2](/docs/graphics/Energy_Manager_Line_KPI_Instance_2.png)
+
+4) no need to change anything in the general display options > Continue
+5) no need to change anyting in the Chart dispaly options > Finish
```
Comment (mattDoerr) [inline docs/Usage.md]: typo: anyting

**Review by mattDoerr — CHANGES_REQUESTED** [review]

File: docs/Usage.md (hunk 1 above)
Comment (EvaKaul) [reply docs/Usage.md]: fixed

File: docs/Usage.md (hunk 2 above)
Comment (EvaKaul) [reply docs/Usage.md]: fixed

File: docs/Usage.md (hunk 4 above)
Comment (EvaKaul) [reply docs/Usage.md]: fixed

File: docs/Usage.md (hunk 5 above)
Comment (EvaKaul) [reply docs/Usage.md]: fixed

**Review by EvaKaul — COMMENTED** [review]
ok

**Review by mattDoerr — APPROVED** [review]

**Pull request merged**